components/yaml.rst

```diff
@@ -250,6 +250,59 @@ representation of the object.
     parsers will likely not recognize the ``php/object`` tag and non-PHP
     implementations certainly won't - use with discretion!
 
+Syntax Validation
+~~~~~~~~~~~~~~~~~
+
+The syntax of YAML contents can be validated through the CLI using the
+:class:`Symfony\\Component\\Yaml\Command\\LintCommand` command.
```

Comment:
Yeah it was expected.. ^^ 
Btw, how would you mention it? I thought about a `caution`